crates/query-engine/translation/src/translation/mutation/delete.rs

```diff
@@ -0,0 +1,111 @@
+use crate::translation::query::values::translate_json_value;
+use query_engine_metadata::metadata;
+use query_engine_sql::sql::ast;
+use std::collections::BTreeMap;
+
+// this can get us `DELETE FROM <table> WHERE column = <column_name_arg>`
+#[derive(Debug, Clone)]
+pub enum DeleteMutation {
+    DeleteByKey {
+        description: String,
+        schema_name: ast::SchemaName,
+        table_name: ast::TableName,
+        by_column: metadata::database::ColumnInfo,
+    },
+}
+
+pub fn translate_delete(
```

Comment:
Given the description of a delete mutation (ie, `DeleteMutation`), and the arguments, output the SQL AST.